sql-schema/212.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM `pollers` WHERE `poller_name` LIKE '%\n';
```

Comment:
I was just about to merge this until I saw this.

I'm not sure of the implications with this so I'm going to take a look at my work install tomorrow.

Reply:
This PR makes a change to pollers where poller_name is just the hostname.  The old would have a line return at the end of the hostname.  This now doesn't.  It results in multiple entries for the same poller in the pollers table.  Which isn't really used anywhere except validation and the poller display.  

I wasn't sure how to clean those up, so I just did this since they will be repopulated next poller run.

Reply:
perhaps I could edit poller wrapper to ignore the line return when checking if it exists and update that record with the new name.

Reply:
@laf I reviewed options and came back to this.
1. Update poller-wrapper.py and librenms-service.py to update the a name if not needed, but that adds unnecessary code in both spots.
2. Do an update to trim the poller_name field.  This could result in duplicate entries in the pollers table.
3. Delete all entries that poler_name ends in a line return, they will be re-added next poller run with the correct poller_name.

3 seems the best to me.

Reply:
Looking at the where it's used, 3 does seem fine